### DISCUSSION TITLE
Feature Request: multiple password fields for single entry

### DISCUSSION THREAD

**Discussion by Larrry** [keepassxreboot/keepassxc#8914] [Ideas]
Sometimes we need more than one password associated with entry, for example, most of crypto exchanges require extra passphases, pin-codes or anti-phishing codes. Another example is a simple credit card, where I need to store both pin-code and CVC.
For now, I keep all information like this in the "Notes" area. It's not always convenient, especially when display is visible to someone else. 
Maybe the quick solution is in adding "hidden notes", not extra password fields?

**Comment by droidmonkey** [comment]
On the advanced section when editing an entry you can add additional attributes. 

#8228 and #863 cover this issue.
